client/app/scripts/superdesk-users/views/user-preferences.html

```diff
@@ -30,6 +33,34 @@
                     <span>{{ datelinePreview | previewDateline:datelineSource}} </span>
                 </div>
             </li>
+
+            <li class="fieldset">
+              <div class="legend">
+                {{ :: preferences['categories:preferred'].category }}
+              </div>
+
+              <div sd-info-item>
+                  <label translate>
+                    {{ :: preferences['categories:preferred'].label }}
+                  </label>
+                  <label translate>
+                    By checking the categories you are interested in, the system
+                    will only display them in a menu for setting the content
+                    item's categories (along with any of its existing categories).
+                  </label>
+
+                  <div ng-repeat="cat in categories track by cat.qcode">

```

Comment:
Actually, on the Authoring screen if you press the **(+)** button, you already see all the categories you can choose from in the menu - at least that's what I see?

The length argument on the settings is valid, I was thinking about in on my way to the office, and there are two points to consider:

* Preferred categories are the very last setting in the tab, thus it's not possible for a user to overlook a setting because it might be hidden too low. There are also not that many of them, it looks fine to me (on my laptop's resolution, though... and it's only a personal opinion).
* Is it possible to trivially implement "select all/none" buttons? I assume it is, it's only a matter of changing the array holding the choices.

Anyway, thanks for the comment, this kind of feedback is always appreciated. I suggest we discuss the matter with  others when seeing the settings tab in action on the screen.